game-frontend/src/components/user-script/editorIntellisense.txt

```diff
@@ -0,0 +1,5 @@
+type Direction = 'UP' | 'DOWN';
```

Comment:
Where is this file used?

Reply:
nowhere yet. the idea was to put the intellisense stuff in a separate file. couldn't make it to work.

Reply:
Then i would suggest you to go the easy way and add the typings in a separate `.ts` file and export them as a string:
```.js
export default `
  add your typings here
`
```

Reply:
How would go about this if the file should be read as a string. The whole file. So we could enable TS intellisense for the file.

Reply:
i get
`All files must be modules when the '--isolatedModules' flag is provided.  TS1208`
when i try that.

Reply:
ok, added `export {}` at the last line, now its working...

Reply:
This last line should not be necessary. Works just fine without that line.

Reply:
mh... TS1208 error is gone now even after removing the line. done.